docs/developer-guide/joins/join-streams-and-tables.md

```diff
@@ -40,6 +40,9 @@ CREATE STREAM pageviews_enriched AS
   EMIT CHANGES;
 ```
 
+> Note: Currently, the join expression must be a simple comparison.
+Non-equi joins, or joins on multiple columns are not supported yet. 
```

Comment:
```suggestion
    or joins on multiple columns are not supported. 
```

Reply:
maybe worth adding a few words to say that if you want multti-col joins your workaround is to `join` on key equality and add a where clause to compare the other relevant cols ?